src/components/AdjustVaultForm.tsx

```diff
@@ -0,0 +1,71 @@
+import ErrorWarning from 'svg/error-warning';
+import AmountInput from './AmountInput';
+import StyledDropdown from './StyledDropdown';
+
+const AdjustVaultForm = () => {
+  return (
+    <div className="bg-white font-serif p-8 shadow-[0_40px_40px_-14px_rgba(116,116,116,0.25)] rounded-[20px] w-full">
+      <div className="font-bold mb-4">Adjust Collateral</div>
+      <div className="mb-12 grid grid-cols-2 gap-10">
+        <div className="text-[#9193A5] text-sm col-span-2 lg:col-span-1">
+          Deposit additional collateral or withdraw your existing collateral.
+          Select No Action to leave collateral unchanged.
+        </div>
+        <div className="col-span-2 lg:col-span-1">
+          <div className="mb-6">
+            <StyledDropdown
+              selection="No Action"
+              choices={['No Action']}
+              onSelection={() => {
+                console.log('Handle Action Selected');
```

Comment:
low risk that you don't get to these but consider in the future making them more loud. E.g. logging as `console.error` and including TODO or FIXME.

Reply:
Added simple TODO, yea this is up next.